content/en/tracing/guide/setting_primary_tags_to_scope.md

```diff
@@ -76,12 +76,14 @@ Note:
 
 * Only organization administrators have access to this page.
 * Changes may take up to two hours to be reflected in the UI.
+* There are some tags that the tracer always adds to spans: `resource`, `name`, and `service`. To avoid potentially confusing UI behavior never add these as host level tags. 
```

Comment:
```suggestion
* The tracer always adds `resource`, `name`, and `service` tags to spans. Datadog recommends never adding these as host level tags to avoid confusion.
```